### ISSUE TITLE
"Warning: Native component for "MKTouchable" does not exist"

### ISSUE THREAD

**Issue by HarrisRobin** [xinthink/react-native-material-kit#53]
Hi!

For some reason, whenever I try to use any of the buttons I get the following error on the console and touch events aren't triggered.

```
Warning: Native component for "MKTouchable" does not exist
```
I am using react-native 0.14.1 and react-native-material-kit 0.2.2
Anyone else running into this issue? 

Thanks :)

**Comment by xinthink** [comment]
Did you link the lib to your project following [the guide](https://github.com/xinthink/react-native-material-kit#getting-started)?

**Comment by xinthink** [comment]
hi @HarrisRobin, did the problem fixed?

**Comment by HarrisRobin** [comment]
@xinthink sorry for the late reply. Let me try again tonight after work and get back to you!

**Comment by HarrisRobin** [comment]
@xinthink i never understood the root cause of this, but from a fresh project and following the guide again it seems to work now. I will re-open if i understand what caused it in the first place!

**Comment by xinthink** [comment]
thanks @HarrisRobin

**Comment by ourway** [comment]
I have this issue, on Android and react-native 1.7.   I tried everything without success

**Comment by xinthink** [comment]
Just tested on RN 0.17.0-rc, it's working fine for me both on iOS and Android.

Please make sure the library is linked to your project (follow the [Getting started guide](https://github.com/xinthink/react-native-material-kit#getting-started))

For android projects, besides the dependency declaration in the gradle build file, you'll also have to register the instance of `ReactMaterialKitPackage` to the `ReactInstanceManager`.

**Comment by HubMahesh** [comment]
Hi there,
I want to use custom segment control in react native
and i found https://recordnotfound.com/react-native-custom-segmented-control-wix-127433
 but showing error:

native component for CustomSegmentedControl does not exist.
Can i get any solution ?

**Comment by JulianKingman** [comment]
I'm having this issue as well. React-native link ran successfully, I can see the package in xcode. I removed `node_modules` and reinstalled but still get the error. Ideas?

**Comment by JulianKingman** [comment]
Nevermind, I went into xcode and tried to build and noticed the usual post-react-native-0.40 build errors (changing "RCTView.h" to <React/RCTView.H>, and the such). Fixing those, building, deleting the app and reinstalling seems to have fixed it.
